tools/config/index.ts

```diff
@@ -287,7 +287,7 @@ if (options.generateDeps) {
   execSync(`npm run generate:deps`);
 } else {
   // collect and generate dependencies.json file.
-  execSync(`npm run generate:deps --compare=true`);
+  execSync(`npm run generate:deps:compare`);
```

Comment:
Ah... so it looks like a regression bug related to switching to `npm` from `yarn`. The functionality was there before and stopped working due to differences in CLi syntax for passing additional parameters to a script.

We can avoid adding a new `generate:deps:compare` script to our `package.json` by adding a tiny change (--) to the previous version of the script invocation in this line:
```suggestion
  execSync(`npm run generate:deps -- --compare=true`);
```
(some history here: https://github.com/npm/npm/pull/5518)